components/prize_pool/commons/prize_pool_placement_base.lua

```diff
@@ -146,6 +164,7 @@ function BasePlacement:parseOpponentArgs(input, date)
 	elseif type(opponentArgs[1]) ~= 'table' then
 		opponentData = Opponent.readOpponentArgs(opponentArgs)
 	end
+	---@cast opponentData standardOpponent
```

Comment:
Need to think if this is correct, especially since we do have a nil check on the next line

Reply:
yeah it can be nil too, adjusted it accordingly